### ISSUE TITLE
Quick Start errors.

### ISSUE THREAD

**Issue by dlaidlaw** [kubermatic/kubermatic#9484]
### What happened?

Using the Quick Start instructions at https://docs.kubermatic.com/kubermatic/v2.20/installation/install_kkp_on_node/, there is a problem with either the instructions or the makefile. The instructions under the heading **Create the Infrastructure using Terraform** are:
```
eval `ssh-agent`
ssh-add /path/to/.ssh/id_rsa
make k1-tf-apply PROVIDER=aws
```

The k1-tf-apply target depends on k1-load-env, which does the eval and ssh-add commands, so they are not required. Plus, the k1-load-env uses and env var named K1_SSH_KEY, which is not set and causes command failures.

Perhaps the instructions are incorrect? Maybe a specific branch or tag should be checked out? The instructions just clone.

### What should happen?

No errors.

Also, an indication of what to set the KKP_DNS variable to would be good. There is no explanation of how this is used. For example, this is deploying into AWS, so is it a DNS server from AWS? Maybe the one for the VPC? 

### How to reproduce?

Follow the quick start instructions as referenced above.

### Environment

- KKP Version: 2.20
- Shared or separate master/seed clusters?: n/a
- Master/seed cluster provider: n/a

**Comment by ahmedwaleedmalik** [comment]
@dlaidlaw Thank you for pointing it out and apologies for the inconvenience. We have updated the [repository](https://github.com/kubermatic/kkp-single-node) and the documentation with the fixes.

`KKP_DNS` would be the domain that you'd want to use for the Kubermatic dashboard, for example, kkpdashboard.com

**Comment by ahmedwaleedmalik** [comment]
I'm closing this issue since it's resolved. But feel free to open up a new issue if you face any other hinderances.
